components/org.wso2.carbon.identity.api.expired.password.identification/org.wso2.carbon.identity.api.expired.password.identification.v1/pom.xml

```diff
@@ -0,0 +1,182 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org).
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
seems to be different from the usual license.

Reply:
Addressed with commit d3d1013c616d84f1e13fc5cb72e42bdb784ff253